tests/Dfe.EarlyYearsQualification.UnitTests/Helpers/CheckServiceAccessKeysHelperTests.cs

```diff
@@ -0,0 +1,102 @@
+using Dfe.EarlyYearsQualification.Web.Helpers;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Moq;
+
+namespace Dfe.EarlyYearsQualification.UnitTests.Helpers;
+
+[TestClass]
+public class CheckServiceAccessKeysHelperTests
+{
+    [TestMethod]
+    public void AllowPublicAccess_ConfigSetAsFalse_ReturnsFalse()
+    {
+        var dic = new Dictionary<string, string?>
+                  {
+                      { "ServiceAccess:IsPublic", "false" }
+                  };
+
+        var configuration = new ConfigurationBuilder()
+                            .AddInMemoryCollection(dic)
+                            .Build();
+
+        var helper = new CheckServiceAccessKeysHelper(configuration);
+
+        var result = helper.AllowPublicAccess;
+
+        result.Should().Be(false);
```

Comment:
Not an issue, but just for future reference, there are also. extensions for .BeFalse() and .BeTrue()